.yarn/patches/@credo-ts-anoncreds-npm-0.5.13-446ac3168e.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/build/anoncreds-rs/AnonCredsRsHolderService.js b/build/anoncreds-rs/AnonCredsRsHolderService.js
+index c1eeeca1c9c254f47880909391b22f66598f42fd..779b88d59841070a5e298e0544908a7521bc1cf2 100644
+--- a/build/anoncreds-rs/AnonCredsRsHolderService.js
++++ b/build/anoncreds-rs/AnonCredsRsHolderService.js
+@@ -82,7 +82,8 @@ let AnonCredsRsHolderService = class AnonCredsRsHolderService {
+                         ].join('\n'));
+                     }
+                 }
+-                const { linkSecretId, revocationRegistryId, credentialRevocationId } = (0, w3cAnonCredsUtils_1.getAnoncredsCredentialInfoFromRecord)(credentialRecord, (0, proofRequest_1.proofRequestUsesUnqualifiedIdentifiers)(proofRequest));
++                const proofUsesUnqualifiedIdentifiers = (0, proofRequest_1.proofRequestUsesUnqualifiedIdentifiers)(proofRequest);
++                const { linkSecretId, schemaId, credentialDefinitionId, revocationRegistryId, credentialRevocationId } = (0, w3cAnonCredsUtils_1.getAnoncredsCredentialInfoFromRecord)(credentialRecord, (0, proofRequest_1.proofRequestUsesUnqualifiedIdentifiers)(proofRequest));
```

Comment:
@jamshale could we not just pass in the var above like this?
```suggestion
+                const { linkSecretId, schemaId, credentialDefinitionId, revocationRegistryId, credentialRevocationId } = (0, w3cAnonCredsUtils_1.getAnoncredsCredentialInfoFromRecord)(credentialRecord, proofUsesUnqualifiedIdentifiers);
```

Reply:
I think this must be a js compiler thing? That's how it is with the ts code.

![image](https://github.com/user-attachments/assets/69270a19-3be1-4c6e-a177-201f159e28e0)